app/controllers/amends/AmendExportedPlasticPackagingController.scala

```diff
@@ -35,4 +36,11 @@ class AmendExportedPlasticPackagingController @Inject()(
     implicit request =>
       Ok(view())
   }
+
+  def onSubmit : Action[AnyContent] = identify.async {
```

Comment:
is this needed? i dont think it is

Reply:
yes, ok it could be done in the scala.html file. I think I just left it as the scaffolding put it there.